modules/tachyon/src/main/tachyon_0.8.2

```diff
@@ -0,0 +1 @@
+tachyon_0.7.1
```

Comment:
why `tachyon_0.7.1` inside a file named `tachyon_0.8.2` ?

also is this `text-file` really intended to live inside `modules/src/main...` ?